torch/nn/modules/module.py

```diff
@@ -1572,10 +1582,16 @@ def _call_impl(self, *args, **kwargs):
 
         return result
 
-    __call__ : Callable[..., Any] = _call_impl
+    __call__ : Callable[..., Any] = _call_impl if torch.jit.is_scripting() else _wrapped_call_impl
```

Comment:
hmmm are we scripting the Module creation? If not, that might not work. But if CI is green, I'm happy!